### DISCUSSION TITLE
Time to have LTS Versions  (Long-Term-Support)

### DISCUSSION THREAD

**Discussion by sinanisler** [elementor/elementor#23843] [Developers Community]
Time to have LTS Versions  
(Long-Term-Support)

some versions has to have LTS support. so we can avoid  updating to latest just for small bug just for small exploit just for that this whatever...

LTS versions would solve this problems. we can pick LTS version to build our client sites and don't think about it anything else long term...
please consider it.

1.18 LTS
1.19
1.20 LTS
...
..

**Comment by sinanisler** [comment]
an example;

![image](https://github.com/elementor/elementor/assets/1686324/1672835e-476b-4938-9cc3-4f1d265f5e21)

there are lots of LTS software examples out there everyone makes its own way of building LTS versions.

**Comment by theavic** [comment]
I'm glad someone is bringing this up, I was worried about this back when Elementor had a serious security issue, and I was just on the tail end of my license. IMO software companies need to roll out security hotfixes to everyone in the interest of the common good - but short of that some kind of LTS would really help.

**Reply by sinanisler** [reply]
yep 
like every other version skiping one version for LTS is very managable.  
example;

1.18 LTS
1.19
1.20 LTS
...
..
